transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoop.java

```diff
@@ -201,7 +209,8 @@ private int epollWait(boolean oldWakenUp) throws IOException {
         long currentTimeNanos = System.nanoTime();
         long selectDeadLineNanos = currentTimeNanos + delayNanos(currentTimeNanos);
         for (;;) {
-            long timeoutMillis = (selectDeadLineNanos - currentTimeNanos + 500000L) / 1000000L;
+            long timeoutMillis = Math.min(MAX_EPOLL_WAIT_TIMEOUT,
```

Comment:
I'm not sure I get your question. Are you asking about `selectDeadLineNanos`? If so, it's to execute a scheduled task.